extensions/amp-ad-network-doubleclick-impl/0.1/test/test-doubleclick-rtc.js

```diff
@@ -271,4 +272,40 @@ describes.realWin('DoubleClick Fast Fetch RTC', {amp: true}, env => {
           .to.deep.equal(response);
     });
   });
+
+  describe('getCustomRealTimeConfigMacros', () => {
```

Comment:
Add test coverage for truncation

Reply:
done, broke out to separate function for easiness of testing